cysync/src/pages/lockscreen/LockScreen.tsx

```diff
@@ -151,6 +151,7 @@ const LockScreen = (props: any) => {
   const ENTER_KEY = 13;
   const handleKeyPress = (event: any) => {
     if (event.keyCode === ENTER_KEY) {
+      setIsLoading(true);
       handleSubmit();
```

Comment:
Remove the handleSubmit call. It will be automatically called after setting the loading to true.

Also, make the same change to remove password, change password and set password.